locals.tf

```diff
@@ -0,0 +1,80 @@
+locals {
+
+  deploy_ucfs_server_stub = {
+    development = true
+    qa          = true
+    integration = false
+    preprod     = false
+  }
```

Comment:
We're controlling whether or not we stub this service via the pipeline (i.e. not even bothering with a prod deployment). So we can rip this local out and any references to it.